src/averageAges.js

```diff
@@ -37,7 +53,29 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let count = 0;
```

Comment:
Use a more descriptive variable name instead of `count`. Consider using `womenCount` for better readability.

Reply:
You might event not need it and use array.length property to access total count of items in resulting array